backend/app/views/spree/admin/users/edit.html.erb

```diff
@@ -23,9 +23,22 @@
         </div>
       <% end %>
     <% end %>
+
   </div>
 </fieldset>
 
+<fieldset id='forgot-password'>
```

Comment:
We should add a `no-border-bottom` class. There are already too much borders on this page, also I don't not see any need for adding a DOM ID.